### ISSUE TITLE
Book

### ISSUE THREAD

**Issue by jennocool** [ItsGabry/NanoCombiner#8]
Can you make it if I do an enchantingbook and an sword that you will get the enchant
And also 2 different enchantments

**Comment by TheNewHEROBRINEX** [comment]
Like you do in the anvil in vanilla?

**Comment by jennocool** [comment]
Yes that it has the same features as An anvil in vanilla with combining
